helm/hpcc/templates/_helpers.tpl

```diff
@@ -1061,11 +1061,30 @@ Pass in dict with root, category.  optional name to restrict it to a single name
 
 {{/*
 Create placement related settings
-Pass in dict with placement
+Pass in dict with me for current placements and dict with new for the new placements
```

Comment:
This is very minor - however this sentence confused me a little - since I don't really understand all these variables, but if this is appropriate language I would rather say something like pass in dict with the me prefix (if that is an appropriate term) for current placements and dict with the new prefix for the new placements